### PR TITLE
[12.0][IMP] connector_equipment

### PR DESCRIPTION
**Pull request by murtuzasaleh**
#29

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@dreispt @max3903

**Review by dreispt — APPROVED** [review]

File: connector_equipment/demo/backend_equipment_demo.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline connector_equipment/demo/backend_equipment_demo.xml]:
```suggestion
```
Not required in version 12...

**Review by max3903 — APPROVED** [review]
LGTM

File: connector_equipment/demo/backend_equipment_demo.xml (hunk 1 above)
Comment (dreispt) [reply connector_equipment/demo/backend_equipment_demo.xml]: I don't have access to commit the suggestion...

**Pull request merged**